dashboard/influx-data-gen/influx-data-gen.lua

```diff
@@ -0,0 +1,87 @@
+--
+-- SPDX-FileCopyrightText: (c) 2024 Ring Zero Desenvolvimento de Software LTDA
+-- SPDX-License-Identifier: GPL-2.0-only
+--
+
+local socket = require("socket")
+local lfs = require("lfs")
+
+local header = 'https,host=ring-0.io,location=rj'
+local flow = 100 -- ~ request/second  (max: 4000)
+
+math.randomseed(os.time())
+
+function read_file_to_table(file_name)
+    local file <close> = io.open(file_name, "r")
+
+    if not file then
+        print("Error opening the file: " .. file_name)
+        return nil
+    end
+
+    local words_table = {}
+    for line in file:lines() do
+        table.insert(words_table, line)
+    end
+
+    file:close()
```

Comment:
I think it's safe to omit this line. `local file <close>` does this automagically upon destruction.